test/e2e/features/pageObjects/caseEditPage.js

```diff
@@ -241,7 +241,7 @@ class caseEditPage {
                     let keyVal = String(date.getDate()).padStart(2, '0') + "-" + String(date.getMonth() + 1).padStart(2, '0') + "-" + date.getFullYear();
                     this.checkURanswerPageData[i][Object.keys(this.checkURanswerPageData[i]).toString()] = keyVal;
                 }
-                if (Object.values(this.checkURanswerPageData[i]).toString() && Object.keys(this.checkURanswerPageData[i]).toString() != "Compliance") {
+                if (Object.values(this.checkURanswerPageData[i]).toString() && Object.keys(this.checkURanswerPageData[i]).toString() != "Compliance" && Object.keys(this.checkURanswerPageData[i]).toString() != "") {
```

Comment:
Probably being over-pedantic but should the string equality be using `!==` (strict) instead of `!=`?

Reply:
Maybe. This is a functional test, though. @sreekanthpuligadda?

Reply:
addressed